README.md

```diff
@@ -47,33 +47,40 @@ If you want to export the functions to the main namespace, simply use the comman
 ```julia
 using VectorizedRoutines.R
 rpois(1,[5.2;3.3]) # Same as R.rpois(1,[5.2;3.3])
+
+```
+The original Julia vectorized routines offered here do not need to be prefaced:
+
+```julia
+xs = 1:26
+pairwise((x,y)->sqrt(x^2 + y^2), xs)
 ```
 
 # Current Functions and Macros
 
 - MATLAB
-  - ngrid
-  - meshgrid
-  - accumarray (fast)
-  - accumarray2 (more functionality)
+  - `ngrid`
```

Comment:
This is actually unrelated, but I saw you'd added backticks for `@vcomp` below so I added them everywhere. Happy to remove again.